app/styles/addons/gh-koenig/gh-koenig.css

```diff
@@ -164,7 +164,7 @@
 }
 .kg-card .kg-card-toolbar button.kg-card-delete {
     text-transform: none !important;
-    font-family: "ghosticons" !important;
+    /*font-family: "ghosticons" !important;*/
```

Comment:
@disordinary I commented this out because I didn't know how it's used and couldn't find it.